atomic_reactor/tasks/binary_container_build.py

```diff
@@ -201,6 +203,7 @@ def build_container(
         build_dir: dirs.BuildDir,
         build_args: Dict[str, str],
         dest_tag: ImageName,
+        flatpak: bool,
```

Comment:
Maybe this could be a `squash_all` param instead? I would rather have the logic of how flatpaks are different from regular containers in the Task itself rather than the PodmanRemote class, that one should really just handle podman remote commands

Reply:
hmm, that seems a better idea. Update to pass squash all.